include/seqan3/io/alignment_file/format_sam.hpp

```diff
@@ -1086,6 +1089,17 @@ inline void format_sam::write_field(stream_t & stream, field_type field_value)
         stream << field_value;
 }
 
+/*!\brief Writes a field value to the stream.
+ * \tparam stream_t           The stream type.
+ * \param[in,out] stream      The stream to print to.
+ * \param[in]     field_value The value to print.
+ */
```

Comment:
If this is always the same documentation, why not doing `\overload`? Like also before?